SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -124,6 +124,26 @@ void USpatialReceiver::OnAddEntity(const Worker_AddEntityOp& Op)
 	UE_LOG(LogSpatialReceiver, Verbose, TEXT("AddEntity: %lld"), Op.entity_id);
 }
 
+void USpatialReceiver::RemoveRedundantRemoveComponentOps(const Worker_AddComponentOp& Op)
+{
+	int32 RemovedOps = 0;
+	for (int32 i = QueuedRemoveComponentOps.Num() - 1; i >= 0; --i)
```

Comment:
This can be replaced with a RemoveAll with a predicate

Reply:
nice